.github/dependabot.yml

```diff
@@ -2,13 +2,11 @@ version: 2
 updates:
   - package-ecosystem: pip
     directories:
-      - /utils/cq/requirements.txt
+      - /utils/cq/
+      - /
```

Comment:
Not sure if you were aware of this one or didn't include it intentionally:

```suggestion
      - /
      - utils/ansible/ftest/
```

Reply:
Overlooked, thanks!